yaksh/documentation/moderator_docs/creating_question.rst

```diff
@@ -19,23 +19,33 @@ Setting up questions
 
 	* **Language** - Programming language on which the question is based.
 
-	* **Active** -  If the question is active to attempt or not.
```

Comment:
The field still exists. It seems this line got deleted, Can you make sure this is added back to the docs!